src/Table/read_fits.cxx

```diff
@@ -131,11 +131,11 @@ void tablator::Table::read_fits (const boost::filesystem::path &path)
           append_column (c.name (), Data_Type::UINT32_LE, array_size);
           break;
         case CCfits::Tlong:
-          /// Tlong and Tulong have indeterminate sizes.  We guess 32 bit.
-          append_column (c.name (), Data_Type::INT32_LE, array_size);
+          /// Tlong and Tulong have indeterminate sizes.  We guess 64 bit.
```

Comment:
I am super confused by this. Surely the underlying library knows the column size and isn't just returning garbage when we ask for values - so why are we guessing?

My first thought was that CCfits saying a column has value type of Tlong (Tulong) means that it has mapped the on-disk column type to long (unsigned long) values in program memory. Under that interpretation, we should map to INT64_LE if `sizeof(long) == 8` and INT32_LE if `sizeof(long) == 4`, i.e. your change is correct on a 64bit system, but not on a 32bit system. (Or at least, it uses an unnecessarily wide type on the latter.) 

However, check out this lovely snippet from the CFITSIO header fitsio.h:
```c
#define TLONG        41
#define TINT32BIT    41  /* used when returning datatype of a column */
```
and this bit of the manual https://heasarc.gsfc.nasa.gov/docs/software/fitsio/c/c_user/node20.html
```
  The following data type code is only for use with fits\_get\_coltype
  #define TINT32BIT    41  /* signed 32-bit int,         'J' */
```

Since `CCFits::ValueType::Tlong` is just `TLONG` from CFITSIO, I think that means we should treat `Tlong` as exactly a 32 bit integer, but _only_ when reading! In other words, I think the original code is correct. 🤦‍♂

Reply:
Just for the record, the guessing wasn't my idea---that comment is in the master.  Yes, my changes do depend on the system.  And I did see that snippet, and at one point I even had a hypothesis about what the comment about fits_get_coltype() meant.  But did you try running the updated convert.sh script from this branch against the master version of the tablator binary?  Unless I did something really stupid with the new tests....